website/thaliawebsite/settings.py

```diff
@@ -683,20 +683,19 @@ def from_env(
 
 # TinyMCE config
 TINYMCE_DEFAULT_CONFIG = {
-    "selector": "textarea",
```

Comment:
This selector is no longer required?

Reply:
Nope, selecting it will select _all_ texture's on a page. The new library will automatically select based on the field id